data/mods/Xedra_Evolved/items/inventor/gunmods.json

```diff
@@ -3,7 +3,7 @@
     "id": "trinket_damage",
     "type": "GUNMOD",
     "name": { "str": "projectile accelerator" },
-    "description": "Small part of unusual shape.  It can be mounted into the end of your barrel, which somehow increases the speed of the projectile.  It is also makes the gun much louder and difficult to hold after each shot.",
+    "description": "A small cylinder, reminescent of a muzzle booster.  It can be mounted onto the end of your barrel, which somehow increases the speed of the projectile.  It also makes the gun much louder and difficult to hold after each shot.",
```

Comment:
```suggestion
    "description": "A small cylinder, reminiscent of a muzzle booster.  It can be mounted onto the end of your barrel, which somehow increases the speed of the projectile.  It also makes the gun much louder and difficult to hold after each shot.",
```